docs/development/codeguide.rst

```diff
@@ -46,15 +46,12 @@ Interface and Dependencies
   :ref:`dev-portable`.
 
 * The new Python 3 formatting style should be used (i.e.
-  ``"{0:s}".format("spam")`` instead of ``"%s" % "spam"``), although
-  when using positional arguments, the position should always be
-  specified (i.e.  ``"{:s}"`` is not compatible with Python
-  2.6).
+  ``"{0:s}".format("spam")`` instead of ``"%s" % "spam"``).
 
 * The core package and affiliated packages should be importable with
   no dependencies other than components already in the Astropy core,
   the `Python Standard Library
-  <http://docs.python.org/release/2.6/library/index.html>`_, and
+  <http://docs.python.org/3.6/library/index.html>`_, and
```

Comment:
https://docs.python.org/3/library/index.html might be more general; that way, you don't have to keep updating the link until Python 4 comes out.